_commands/tools/sftp.md

```diff
@@ -3,22 +3,152 @@
 
 sftp
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
-
-<!-- minimal example -->
+`sftp` is used to securely transfer files between a local machine and a remote machine that is accessed via SSH. SFTP stands for Secure File Transfer Protocol.
 ~~~ bash
-*short* example of command usage and output
+$ sftp user@hostname
+Password: *********
+Connected to hostname.
+sftp> put filename.txt
+sftp> mput file1.txt file2.txt file3.txt
+sftp> get file.txt
+sftp> mget file1.txt file2.txt file3.txt
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `sftp user@hostname`
+~~~ bash
+$ sftp mmdarden@login.engin.umich.edu
+Password: **********
+Connected to login.engin.umich.edu.
+sftp>
+~~~
+
+##### Break it down
```

Comment:
Replace these placeholder headings with meaningful headings, or remove them if not needed.